File: 1456 maximum-number-of-vowels-in-a-substring-of-given-length/solution.cs
public class Solution {
    public int MaxVowels(string s, int k) {
        int count=0;
        bool[] v = new bool[126];
        int max=0;
        v['a']=v['e']=v['i']=v['o']=v['u']=true;
        for(int i=0;i<s.Length;i++) {
            if(v[s[i]]) count++;
            if(i>=k && v[s[i-k]]) count--;
            max=Math.Max(max,count);
        }
        return max;
    }
}

File: 1456 maximum-number-of-vowels-in-a-substring-of-given-length/solution.go
func maxVowels(s string, k int) int {
    var v [126]bool
    v['a']=true
    v['e']=true
    v['i']=true
    v['o']=true
    v['u']=true
    c,m:=0,0
    for i:=0;i<len(s);i++ {
        if v[s[i]] {
            c+=1
        }
        if i>=k && v[s[i-k]] {
            c-=1
        }
        if c>m {
            m=c
        }
    }
    return m
}
